lib/workers/repository/process/extract-update.ts

```diff
@@ -96,8 +113,8 @@ export async function extract(
       logger.info({ err }, 'Error deleting cached dep updates');
     }
   } else {
-    await checkoutBranch(baseBranch!);
```

Comment:
This change seems important. Was it unnecessary, or has it been replaced?

Reply:
It was a mistake.